cci-stats/cmd/runner/main.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
It would be great to have a short comment, or readme, describing what `cci-stats` is/does.